hack/release.sh

```diff
@@ -377,7 +371,7 @@ main() {
 	setup_s3
 	setup_gpg
 	release_binaries
-	release_ubuntu
+	release_repos
```

Comment:
this should not change for 1.9, we need to release the old repos still, hence why my branch is just hanging out til then